config-connector/solutions/iam/kpt/service-account/README.md

```diff
@@ -0,0 +1,102 @@
+Service Account
+==================================================
+
+# NAME
+
+  service-account
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create a service account in your desired project, and grant it a specific role (default to iam.serviceAccountKeyAdmin) on the service account resource level.
+
+  You can also create any number of service accounts and grant them any number of roles by adding new or updating the config YAMLs and then applying them.
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
+
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/member-iam member-iam
+  ```
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using a service account
+  (`cnrm-system@[PROJECT_ID].iam.gserviceaccount.com`) with the following
+  roles in your desired project (it doesn't need to be the project where you
+  installed Config Connector):
+
+  - roles/resourcemanager.projectIamAdmin
+  - roles/iam.serviceAccountAdmin
+
+# USAGE
+
+  Replace `${PROJECT_ID?}` with your desired project ID value from
+  within this directory:
+
+  ```
+  kpt cfg set . project-id VALUE
+  ```
+
+  Once the project ID is set in the configs, simply apply the YAMLs:
+
+  ```
+  kubectl apply -f .
+  ```
+
+## OPTIONAL WORKFLOW
+
+  Optionally, you may want to create a different service account, grant a
+  different role to the service account, or grant multiple roles to the
+  service account, etc. Here are the instructions.
+
+**Create a Different Service Account**
+
+  Change the service account name and apply the YAMLs:
+
+  ```
+  kpt cfg set . service-account-name VALUE
+  kubectl apply -f .
+  ```
+
+**Grant a Different Role**
+
+  Change the role([predefined GCP IAM roles](https://cloud.google.com/iam/docs/understanding-roles#predefined_roles)) and apply the YAMLs. Please make sure to delete the iam policy created previously:
+
+  ```
+  kubectl delete -f .
+  kpt cfg set . role VALUE
+  kubectl apply -f .
+  ```
+
+**Grant Multiple Roles**
+
+  Change the KCC resource name and the role each time before you grant a new
+  role to the service account, also please make sure you delete the iam policy created previously:
```

Comment:
I don't see how this subworkflow supports "grant multiple roles" scenario.

* By doing `kubectl delete -f .`, you clean up everything i.e. the old service account and old iam policy. After you apply the new yamls, a new service account will be created and only the role VALUE_2 is granted. This is not granting **multiple** roles.
* Even if you don't delete the old resources, due to the nature of IAM Policy, the old bindings specified in the old yamls will be overwritten by the new bindings, so only the latest role VALUE_2 will be kept in the iam policy. And it still does not grant **multiple** roles.

Reply:
This section is no longer necessary in the README.